### DISCUSSION TITLE
EMBY中添加带台标和指南数据的源

### DISCUSSION THREAD

**Discussion by sumingyd** [billy21/Tvlist-awesome-m3u-m3u8#53] [Show and tell]
### **EMBY安装插件：XmlTV**

1. 管理员打开EMBY后台设置
2. 找到“电视直播”
3. 选择“添加指南数据源”，区域设置“China”，指南源设置“XmlTV”
4. 下一步，文件或URL 填写：“[http://epg.51zmt.top:8000/e.xml](http://epg.51zmt.top:8000/e.xml )”    ，其他保持默认。

### **IPTV源文件处理**

1. [点此下载](https://github.com/imDazui/Tvlist-awesome-m3u-m3u8)一个直播源
2. 将下载下来的文件上传到[这里](http://epg.51zmt.top:8000/)（这个系统会根据name自动匹配台标和tvg-id）
3. 将自动处理完毕的文件下载下来，并添加到EMBY中的电视源中。

**Comment by leeerogers** [comment]
thanks

**Comment by weiliu2017** [comment]
可以用，谢谢!
